src/aspire/source/image.py

```diff
@@ -85,6 +90,13 @@ def __init__(self, L, n, dtype="double", metadata=None, memory=None):
 
         logger.info(f"Creating {self.__class__.__name__} with {len(self)} images.")
 
+    @property
+    def n_ctf_filters(self):
```

Comment:
This should just count the number of unique filters. Whether they're instances of `CTFFilter` or not shouldn't matter. As long as there are filters, we can call `filter.sign`, which means we can perform phase flip. It's not specific to CTFs.

Reply:
Ok.